dockers/docker-platform-monitor/docker_init.j2

```diff
@@ -42,8 +42,8 @@ if [ -e /usr/share/sonic/platform/platform_wait ]; then
 fi
 
 # If the Python 2 sonic-platform package is not installed, try to install it
-python2 -c "import sonic_platform" > /dev/null 2>&1 || pip2 show sonic-platform > /dev/null 2>&1
-if [ $? -ne 0 ]; then
+PYTHON2_PLATFORM_PACKAGE=/usr/local/lib/python2.7/dist-packages/sonic_platform
```

Comment:
The package can be anywhere in PYTHONPATH. Suggest to use pkgutil.

Reply:
Yes, the paths can change, and pkgutil should be better tool here.

Also, will the new check of path existence validate both `sonic_platform` and `sonic-platform` availability? The previous check was doing that, and I see only `sonic_platform` in new check.